services/m3aggregator/config/aggregator.go

```diff
@@ -276,6 +275,37 @@ func (c *AggregatorConfiguration) NewAggregatorOptions(
 	maxAllowedForwardingDelayFn := c.Forwarding.MaxAllowedForwardingDelayFn(jitterEnabled, maxJitterFn)
 	opts = opts.SetMaxAllowedForwardingDelayFn(maxAllowedForwardingDelayFn)
 
+	// Set whether to enable eager forwarding.
+	if c.Forwarding.EnableEagerForwarding != nil {
```

Comment:
move these logic to a method on `forwardingConfiguration` struct?

Reply:
Hm yeah I can try.